ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/ValidatedCommandFactory.java

```diff
@@ -212,11 +214,22 @@ private static Command createForPlannedQuery(
   ) {
     final KsqlPlan plan = context.plan(serviceContext, statement);
 
-    final ConfiguredKsqlPlan configuredPlan = ConfiguredKsqlPlan
+    ConfiguredKsqlPlan configuredPlan = ConfiguredKsqlPlan
         .of(plan, statement.getSessionConfig());
 
-    context.execute(serviceContext, configuredPlan);
-
+    final KsqlExecutionContext.ExecuteResult result = context
+        .execute(serviceContext, configuredPlan);
+    if (result.getQuery().isPresent()
```

Comment:
Could we just simplify this as 

```
if (result.getQuery().get() instanceof PersistentQueryMetadataImpl) // always override the config to false
```
?

Reply:
The get without the is present check leaves us open to an no such element exception. I would rather just have the check than have to set up error handling logic. 

As for always setting the flag to be false. We can conceivably do this, but I don't really want to add an unrelated override to queries that are not already using the FF. I am just not sure about the consequences and figured it would be safer to just do the check. If you have a strong opinion on this I have no problem modifying it :)

Reply:
No strong opinions, was just wondering the motivation for being very specific. All good now.